### ISSUE TITLE
Exiting fullscreen mode breaks layout in Firefox

### ISSUE THREAD

**Issue by stefankp** [froala/wysiwyg-editor#3307]
Exiting the fullscreen mode sets margins and paddings on the containing elements to 0 with inline styles. This breaks the layout, as seen on the screenshots below from the Froala page.

![screen shot 2019-02-18 at 13 17 37](https://user-images.githubusercontent.com/3661203/52950783-7697c880-3380-11e9-97bb-c5dd7db61c5b.png)
![screen shot 2019-02-18 at 13 17 47](https://user-images.githubusercontent.com/3661203/52950784-7697c880-3380-11e9-8261-66afb59392a3.png)
![screen shot 2019-02-18 at 13 17 51](https://user-images.githubusercontent.com/3661203/52950785-77305f00-3380-11e9-84a5-75df1cbb531c.png)

##### Expected behavior.
Exiting fullscreen mode should add inline styles to remove margins/paddings in Firefox.

##### Actual behavior.
Sets margins/paddings on the containing elements to 0 with inline styles in Firefox.
```
<section class="content-4 no-padding-bottom" id="editor-main-demo" style="margin: 0px; padding: 0px;">
  <div class="container" style="margin: 0px; padding: 0px;">
  ...
```

##### Steps to reproduce the problem.
Open https://www.froala.com/wysiwyg-editor with Firefox and toggle fullscreen on and off.

##### OS.
Mac OS 10.14.3

##### Browser.
Firefox 65

**Comment by craigjennings11** [comment]
Is there a release scheduled that will contain this fix?

**Comment by stefanneculai** [comment]
Yes, it will be part of the next release.

**Comment by jhubert** [comment]
Confirmed that this is fixed in master. We are currently running the minified version of fullscreen.js from the master branch until 2.9.4 is released.

**Comment by sumancholleti** [comment]
Hi, So is this fixed in 2.9.4 ? if not please let me know whats the work around
